api/src/db/migrations/migrations.ts

```diff
@@ -284,4 +285,7 @@ export const Migrations: MigrationFunction[] = [
   migrate_v137_to_v138,
   migrate_v138_to_v139,
   migrate_v139_to_v140,
+  migrate_v140_to_v141,
 ];
+
+export { generatev141UserData as CURRENT_GENERATOR } from "@db/migrations/v141_sync_migrated_and_newly_created_users";
```

Comment:
Updating the documentation around this: when creating a new migration, in addition to adding it to the Migrations const, you should also update what is getting exported as the `CURRENT_GENERATOR`.

Reply:
Could we add a test that checks if `CURRENT_GENERATOR` returns a user with the current version?

Reply:
Do we have a doc on how to do migrations yet?